packages/core/nuxt-theme-module/README.md

```diff
@@ -0,0 +1,43 @@
+# Nuxt Theme Module
+This module:
+- Changes `.nuxt` buildDir to name you set in the config or `.theme` 
+- Adds `reset.scss` transpiled to CSS to the head (it removes default stylings applied by browser, e.g. blue links)
```

Comment:
this should be moved to integration theme tbh, please remove from the doc as we will get rid of that anyway

I already made an issue for that: https://github.com/DivanteLtd/vue-storefront/issues/4702